jest/jest-replace-attribute-snapshot-serializer.ts

```diff
@@ -33,6 +33,13 @@ export function replaceAttributesSnapshotSerializer(
         }
       }
 
+      // Removes “data-v-*” attributes.
+      for (const attribute of value.attributes) {
+        if (attribute.name.startsWith('data-v-')) {
+          value.removeAttribute(attribute.name)
+        }
+      }
+
```

Comment:
I understood that this is removing the data-v attributes? Wasn't sure whether i should still be seeing them in the snapshots? On a brief scan of the diff here, it looks like they've been removed from some places but not others?

Reply:
I'm struggling a bit with getting the custom serializer to go through all nodes. I got most of them removed locally but there are still a handful left. Fixing that next week.

Reply:
Awesome, sounds great! Will be good once they are all gone

Reply:
Figured it out. Much happier with removing those attributes now. This should minimize the amount of times we have to regenerate these snapshots.